handler/handler.go

```diff
@@ -7,18 +7,23 @@ import (
 )
 
 // CreateNATSQueue ready for asynchronous processing
-func CreateNATSQueue(address string, port int, clusterName string, clientConfig NATSConfig) (*NATSQueue, error) {
+func CreateNATSQueue(address string, port int, clusterName, channel string, clientConfig NATSConfig) (*NATSQueue, error) {
 	var err error
 	natsURL := fmt.Sprintf("nats://%s:%d", address, port)
 	log.Printf("Opening connection to %s\n", natsURL)
 
 	clientID := clientConfig.GetClientID()
 
+	// If 'channel' is empty, use the previous default.
```

Comment:
Is "" a valid channel? What about throwing an error for an empty value?

Reply:
This mostly has backwards-compatibility in mind. If for some reason no value for `channel` is provided (i.e. the `faas_nats_channel` environment variable is not defined), it will fall back to the previous (well, current) default without erroring out.